src/main/java/dinkplugin/notifiers/LootNotifier.java

```diff
@@ -42,8 +42,10 @@ public void onNpcLootReceived(NpcLootReceived event) {
     }
 
     public void onPlayerLootReceived(PlayerLootReceived event) {
```

Comment:
this event triggers on player deaths that had nothing to do with our local player...

we could add a setting to only care about likely PK's our local player does (by looking at damage hitsplats)